test/server/lib/Scim.ts

```diff
@@ -572,8 +605,18 @@ describe('Scim', () => {
           });
         });
 
-        checkCommonErrors('patch', '/Users/1', validPatchBody('new name2'));
-      });
+        it('should return 403 when the user is not of type login', async function () {
+          const serviceUserId = await getOrCreateUserId('alfred', {type: 'service'});
+          const res = await axios.patch(scimUrl(`/Users/${serviceUserId}`), validPatchBody('whatever'), chimpy);
+          assert.deepEqual(res.data, {
+            schemas: [ 'urn:ietf:params:scim:api:messages:2.0:Error' ],
+            status: '404',
+            detail: 'User with ID 9 not found'
```

Comment:
```suggestion
            detail: `User with ID ${serviceUserId} not found`
```

Otherwise we are dependent of the other tests run.